src/shared/components/executiveStaffCard/executiveStaffCard.css

```diff
@@ -0,0 +1,62 @@
+.executiveStaffCard {
+  border-radius: 4px;
+  align-items: center;
+  text-align: center;
+  margin: 20px 10px;
+  padding: 20px 10px 10px 10px;
+  width: 280px;
+  height: 100px;
+  background: #F0F2F2;
+  box-shadow: 0 3px 6px 1px rgba(0, 0, 0, 0.1);
+  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
```

Comment:
Let's ditch all the animations and just set height to a greatest common denominator. There's not enough content to justify hiding/unhiding on hover. It'll just look a bit cleaner.